manifests/4.4/elasticsearch-operator.v4.4.0.clusterserviceversion.yaml

```diff
@@ -195,6 +249,10 @@ spec:
                       value: "quay.io/openshift/origin-oauth-proxy:latest"
                     - name: ELASTICSEARCH_IMAGE
                       value: "quay.io/openshift/origin-logging-elasticsearch5:latest"
+                    - name: KIBANA_IMAGE
+                      value: "quay.io/openshift/origin-logging-kibana5:latest"
+                    - name: OAUTH_PROXY_IMAGE
```

Comment:
We have this twice so we can remove one.  Also, the `image-references` file needs to be updated like https://github.com/openshift/cluster-logging-operator/blob/master/manifests/4.4/image-references#L13-L16 to add the kibana image